demo/vl-checkbox.html

```diff
@@ -67,22 +67,24 @@ <h2>Checkbox disabled</h2>
             <pre>
                 <code class="line-numbers language-markup">
                     &lt;vl-checkbox label="Optie 4" disabled&gt;&lt;/vl-checkbox&gt;
+                    &lt;vl-checkbox label="Optie 5" checked disabled&gt;&lt;/vl-checkbox&gt;
                 </code>
             </pre>
             <div class="demo">
-                <vl-checkbox id="checkbox-disabled" label="Optie 4" disabled></vl-checkbox>
+                <vl-checkbox id="checkbox-disabled-uitgevinkt" label="Optie 4" disabled></vl-checkbox>
+                <vl-checkbox id="checkbox-disabled-aangevinkt" label="Optie 5" checked disabled></vl-checkbox>
             </div>
         </div>
 
         <div>
             <h2>Checkbox single</h2>
             <pre>
                 <code class="line-numbers language-markup">
-                    &lt;vl-checkbox label="Optie 5" single&gt;&lt;/vl-checkbox&gt;
+                    &lt;vl-checkbox label="Optie 6" single&gt;&lt;/vl-checkbox&gt;
                 </code>
             </pre>
             <div class="demo">
-                <vl-checkbox id="checkbox-single" label="Optie 5" single></vl-checkbox>
+                <vl-checkbox id="checkbox-single" label="Optie 6" single></vl-checkbox>
```

Comment:
just out of curiosity, wat doet dat attribuut "single"?

Reply:
op de demo pagina komt het label daarvan er niet door trouwens?

Reply:
dat is wat het doet :-)

Reply:
dan is dat een zeer misleidende naam